addons/mail/static/src/components/rtc_option_list/rtc_option_list.xml

```diff
@@ -8,10 +8,18 @@
                     <i class="o_RtcOptionList_button_icon fa fa-lg fa-th-large"/>
                     <span class="o_RtcOptionList_button_text">Change layout</span>
                 </button>
-                <button class="o_RtcOptionList_button" t-on-click="rtcOptionList.onClickFullScreen">
-                    <i class="o_RtcOptionList_button_icon fa fa-lg fa-arrows-alt"/>
-                    <span class="o_RtcOptionList_button_text">Full screen</span>
-                </button>
+                <t t-if="rtcOptionList.rtcController.callViewer.isFullScreen">
+                    <button class="o_RtcOptionList_button" t-on-click="rtcOptionList.onClickDeactivateFullScreen">
+                        <i class="o_RtcOptionList_button_icon fa fa-lg fa-arrows-alt"/>
+                        <span class="o_RtcOptionList_button_text">Deactivate full screen</span>
```

Comment:
Use "disable" as word instead?

Reply:
Maybe only in technical name, the text it would be strage. Youtube uses "Exit full screen" here, and uses a different icon.